packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.tsx

```diff
@@ -12,274 +12,288 @@ import FormSubHeader from '../../../Components/form-sub-header';
 import FormBodySection from '../../../Components/form-body-section';
 import Card from '../../../Containers/proof-of-ownership/card';
 import { IDENTIFIER_TYPES, VALIDATIONS } from '../../../Constants/poo-identifier';
+import { TPaymentMethod, TPaymentMethodInfo } from 'Types';
+import useQuery from '@deriv/api/src/useQuery';
 
 type TProofOfOwnershipForm = {
-    citizen?: string | null;
-    grouped_payment_method_data: any;
+    citizen: string;
+    client_email: string;
+    grouped_payment_method_data: Partial<Record<TPaymentMethod, TPaymentMethodInfo>>;
+    refetchAccountStatus: () => void;
 };
 
 const getScrollOffset = (items_count = 0) => {
     if (isMobile()) return '200px';
     if (items_count <= 2) return '0px';
     return '80px';
 };
-const ProofOfOwnershipForm = observer(({ grouped_payment_method_data }: TProofOfOwnershipForm) => {
-    const { client, notifications } = useStore();
-    const { email: client_email, updateAccountStatus } = client;
-    const { refreshNotifications } = notifications;
+const ProofOfOwnershipForm = observer(
+    ({ client_email, grouped_payment_method_data, refetchAccountStatus }: TProofOfOwnershipForm) => {
```

Comment:
Why is client email passed as props? Can it be fetched from `useCurrentAccountDetails`? Then u can remove the prop from `proof-of-ownership`

Reply:
email can't be fetched from this hook